cleverhans/attacks.py

```diff
@@ -749,6 +749,91 @@ def parse_params(self, y=None, y_target=None, nb_classes=None,
         self.clip_max = clip_max
 
 
+class DeepFool(Attack):
+
+    """
+    DeepFool is an untargeted & iterative attack which aims at finding the
+    minimum adversarial perturbations in deep networks. The implementation
```

Comment:
This makes it hard to understand what the approach is doing. How about something like: "DeepFool is an untargeted & iterative attack which is based on an iterative linearization of the classifier. "

Reply:
Thanks for your description. We have changed to that